homeassistant/components/fronius/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "fronius",
   "name": "Fronius",
   "documentation": "https://www.home-assistant.io/integrations/fronius",
-  "requirements": ["pyfronius==0.5.3"],
+  "requirements": ["pyfronius==0.5.5"],
```

Comment:
This closes  https://github.com/home-assistant/core/issues/54258 right?

Reply:
only https://github.com/home-assistant/core/issues/54258#issuecomment-897282906, but yeah the remainder of the issue has been fixed by another PR